apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/ResourceNames.cs

```diff
@@ -1,10 +1,10 @@
-// Copyright 2017, Google LLC All rights reserved.
+// Copyright 2018 Google LLC
```

Comment:
Did you mean to leave this file in this PR?
(It doesn't really matter one way or the other)

Reply:
Not particularly - but as you say, it doesn't really matter. I might as well leave it in :)